tools/packaging/release/README.md

```diff
@@ -34,7 +27,7 @@ $ ./update-repository-version.sh -h
 To update project version for Kata Containers, use the following:
 
 ```bash
-$ make bump-kata-version NEW_VERSION=<new-version>
+$ make bump-kata-version NEW_VERSION="${new_version}"
```

Comment:
`${new_version}` let image it is a variable, but they are no assignment in this document.

Reply:
Fixed.